source/test/java/com/restfb/types/PostTest.java

```diff
@@ -294,4 +297,30 @@ public void checkV2_8_feedTargetingScheduleUntil() {
     Date ts = examplePost.getFeedTargeting().getRelevantUntilTs();
     assertEquals(1457620560000L, ts.getTime());
   }
+
+  @Test
+  public void checkV2_5_feedTargetingWithRegions() {
+    Post examplePost = createJsonMapper().toJavaObject(jsonFromClasspath("v2_5/post-with-feedtargeting-regions"), Post.class);
```

Comment:
I can provide you an solution this evening. A short test is successful. But I wonder why only the `regions` key is affected ... `cities` and `countries` are OK. BTW is your Post a newly created one? My example is rather old. Perhaps there is a difference, too. However a workaround is on its way :smile: